MonoZelda/Link/Projectiles/Projectile.cs

```diff
@@ -0,0 +1,104 @@
+﻿using PixelPushers.MonoZelda.Sprites;
+using Microsoft.Xna.Framework;
+using PixelPushers.MonoZelda.Commands;
+using System;
+
+namespace PixelPushers.MonoZelda.Link.Projectiles;
+
+public class Projectile
```

Comment:
I think this class might be better called something like `Weapon` or `EquipedWeapon`.